opentelemetry-api/tests/test_implementation.py

```diff
@@ -26,7 +26,12 @@ class TestAPIOnlyImplementation(unittest.TestCase):
     """
 
     def test_tracer(self):
-        tracer_source = trace.TracerSource()
+        with self.assertRaises(TypeError):
+            # pylint: disable=abstract-class-instantiated
```

Comment:
Do you need the pylint and mypy pragma in this file? Lint passes for me without it.

Reply:
Removing the pylint pragma I get:

`opentelemetry-api/tests/test_implementation.py:30:12: E0110: Abstract class 'TracerSource' with abstract methods instantiated (abstract-class-instantiated)`

Removing the `mypy` ignore shows me:
`opentelemetry-api/tests/test_implementation.py:31: error: Cannot instantiate abstract class 'TracerSource' with abstract attribute 'get_tracer'`